electron/legendary/games.ts

```diff
@@ -263,11 +263,8 @@ class LegendaryGame extends Game {
    * @public
    */
   public async addDesktopShortcut(fromMenu?: boolean) {
-    if (process.platform !== 'linux'){
-      return
-    }
     const gameInfo = await this.getGameInfo()
-    const desktopFolder = `${home}/Desktop/${gameInfo.title}.desktop`
+    const desktopFolder = `${app.getPath('desktop')}/${gameInfo.title}.desktop`
```

Comment:
Windows uses the same extension as linux for shortcuts?

Reply:
Oops!